mm2src/coins_activation/src/tendermint_with_assets_activation.rs

```diff
@@ -64,18 +152,37 @@ impl PlatformWithTokensActivationOps for TendermintCoin {
     async fn enable_platform_coin(
         _ctx: MmArc,
         ticker: String,
-        _coin_conf: Json,
+        coin_conf: Json,
         activation_request: Self::ActivationRequest,
         protocol_conf: Self::PlatformProtocolInfo,
         priv_key: &[u8],
     ) -> Result<Self, MmError<Self::ActivationError>> {
-        TendermintCoin::init(ticker, protocol_conf, activation_request, priv_key).await
+        let avg_block_time = coin_conf["avg_block_time"].as_i64().unwrap_or(0);
+
+        // Should be at least bigger than 0 and smaller than 255(u8::MAX)
+        if 1 > avg_block_time || avg_block_time > std::u8::MAX as i64 {
```

Comment:
I got a bit confused, as the code doesn't match the comment 🙂  Should it be `avg_block_time <= 0` or `avg_block_time < 1`?

Reply:
@ozkanonur The condition is `avg_block_time < 1`, but the comment above still states `Should be at least bigger than 0`. Should the comment be modified?